src/com/comeet/data/DataRepository.java

```diff
@@ -97,10 +103,65 @@ public Room retrieveRoomMetadata(String email) {
 
         } catch (Exception e) {
             // e.printStackTrace();
-            ApiLogger.logger.log(Level.SEVERE, "Error getting rooms from database database", e);
+            //ApiLogger.logger.log(Level.SEVERE, "Error getting rooms from database database", e);
```

Comment:
Ok that make sense.  I'm worried that we lose context about just what was going on when the exception was thrown.  I'd rather over log than under log any day.